### PR TITLE
Run 5539 nwi tutorials

### PR DESCRIPTION
**Pull request by dhamberlin**
#### Description of Change
Jira: https://appoji.jira.com/browse/RUN-5539

Provides tutorials for ExternalWindow methods. Changes the docs to point to properly namespaced tutorials.

Also includes a couple light language updates on Window documentation for better clarity.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] relevant documentation is changed or added
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR has assigned reviewers

#### Release Notes

Notes:

### CODE REVIEW

File: src/api/external-window/external-window.ts [hunk 1]
```diff
@@ -372,18 +376,19 @@ export class ExternalWindow extends EmitterBase<ExternalWindowEvents> {
...
      * @experimental
-     * @tutorial Window.stopFlashing
+     * @tutorial ExternalWindow.stopFlashing
      */
     public async stopFlashing(): Promise<void> {
         await this.wire.sendAction('stop-external-window-flashing', this.identity);
     }
 
     /**
+     * NOTE: Group methods on `ExternalWindow`s are currently unstable. This functionality will be addressed in a future update.
```
Comment (licui3936) [inline src/api/external-window/external-window.ts]: UpdateOptions is not group method. Why need this note?

Comment (dhamberlin) [reply src/api/external-window/external-window.ts]: Good catch. Thanks. Overzealous copy-pasting. That said, this method is actually not useful at the moment for two reasons:
1 - It doesn't actually work.
2 - The current set of ExternalWindow options are things we don't actually control without the injection dll, which we decided not to ship. (opacity, showTaskbarIcon, etc)

I'll remove this line now, but not completely sure what the best move is re: ExternalWindow options in general. Do you think we should remove the `update` method from the API?

I realize this is probably out of scope for this PR. Making a note to follow up on this tomorrow.

Comment (licui3936) [reply src/api/external-window/external-window.ts]: probably add @ignore comment in jsdoc for this method if it's not working now.

**Review by tomer-openfin — APPROVED** [review]
lgtm

**Comment by dhamberlin** [comment]
Moved and merged

**Pull request closed without merging**